controllers/articles.controller.js

```diff
@@ -1,14 +1,24 @@
 const {
-    selectArticleById
-} = require("../models/articles.model")
+  selectArticleById,
+  updateArticleById,
+} = require("../models/articles.model");
 
 exports.getArticleById = (req, res, next) => {
-    let id = req.params.article_id;
-    selectArticleById(id)
-      .then((article) => {
-        res.status(200).send({ article: article[0] });
-      })
-      .catch((err) => next(err))
-    
-}
+  let id = req.params.article_id;
+  selectArticleById(id)
+    .then((article) => {
+      res.status(200).send({ article: article[0] });
```

Comment:
This will work, although I would prefer it if you returned rows[0] from the model, because article[0] doesn't make sense semantically and it's also best to do all of our manipulation in the model where possible - the controller should just deal with sending off that response.

Reply:
You can do this ahead of your next PR, if preferred.